package.json

```diff
@@ -1,33 +1,32 @@
 {
   "name": "@4c/graphql-subscription-server",
-  "version": "0.9.0",
-  "files": [
-    "lib",
-    "es"
-  ],
+  "version": "1.0.0",
   "main": "lib/index.js",
   "modules": "es/index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/4Catalyzer/graphql-subscription-server.git"
+  },
+  "author": "4Catalyzer",
+  "license": "MIT",
   "scripts": {
-    "build": "npm run clean && npm run build:cjs && npm run build:esm && npm run build:flow",
-    "build:cjs": "babel src -d lib --ignore __tests__",
-    "build:esm": "BABEL_ENV=esm babel src --out-dir es --ignore __tests__",
-    "build:flow": "for file in $(find ./src -name '*.js' -not -path '*/__tests__*'); do cp \"$file\" `echo \"$file\" | sed 's/\\/src\\//\\/lib\\//g'`.flow; cp \"$file\" `echo \"$file\" | sed 's/\\/src\\//\\/es\\//g'`.flow; done",
-    "check": "flow check",
-    "clean": "rm -rf lib es",
-    "format": "eslint . --fix && prettier --write --ignore-path .eslintignore '**/*.{json,css,md}'",
-    "lint": "npm run check && eslint . && prettier --list-different --ignore-path .eslintignore '**/*.{json,css,md}'",
-    "prepublish": "npm run build",
+    "build": "4c build src --esm",
```

Comment:
@jquense will the `esm` flag build both esm and cjs?

Reply:
it will but you don't need to specify it. It will infer to from the `module` entry